Extensions/outbox.js

```diff
@@ -196,19 +198,15 @@ XKit.extensions.outbox = new Object({
 				m_messages_array = [];
 			}
 
-			m_username = document.location.href.substring(document.location.href.indexOf('/ask_form') + 10);
-			m_username = m_username.substring(0, m_username.indexOf("."));
-
 			var m_obj = {};
 			m_obj.avatar = "ask";
-			m_obj.username = $("#from").find(".tumblelog_name").html();
-			m_obj.message = $("#question").val();
+			m_obj.username = $form.find('.ask-from span:visible').text();
```

Comment:
the username is used when rendering asks on 490. putting "anonymous" here will break it.